### ISSUE TITLE
[Feature Request] Playback loop from 2 specific frames

### ISSUE THREAD

**Issue by 3FiresOnTheLeft** [NickeManarin/ScreenToGif#1244]
Would it be possible to add a section to the playback tab that allows you to give it 2 frame numbers and then loops playback from lowest specified frame to the highest one.

alternatively something like a check box to loop through selected frames only

I often use your program to create good loops it would be useful to be able to test out new start and end points without having to delete frames then undoing it and testing a new loop.

also may be asking too much but I would love to be able to see 2 frames side by side somehow

Thanks for the great program

**Comment by pawlos** [comment]
Would that work if `Playback`->`Play` would just play the selected frames instead of displaying the whole? It does start from selected frame but after reaching the end it loops from the start.

**Comment by 3FiresOnTheLeft** [comment]
Yeah that would make it a lot easier to test the best start and end points to make a good loop
